packages/core-extensions/meta-inputs/meta-haes-rating/src/EditMeta.vue

```diff
@@ -79,37 +113,72 @@
 import { computed, ref } from 'vue';
 
 import isEmpty from 'lodash/isEmpty';
+import cloneDeep from 'lodash/cloneDeep';
 import { haesParams } from './utils';
 import RadarChart from './RadarChart.vue';
-import RatingSliders from './RatingSliders.vue';
+import RateDialog from './RateDialog.vue';
 
 interface Props {
   meta: any;
   dark?: boolean;
   isNew?: boolean;
+  isReviewer?: boolean;
 }
 
 const props = withDefaults(defineProps<Props>(), {
   dark: false,
   isNew: false,
+  isReviewer: false,
 });
 const emit = defineEmits(['update']);
 
 const isEditing = ref(props.isNew);
-const input = ref({ ...props.meta.value ?? {} });
+const input = ref(cloneDeep(props.meta.value ?? { rating: {} }));
+const dialogData = ref(cloneDeep(input.value));
+const isDialogVisible = ref(false);
 
+const reviewRequested = computed(() => input.value?.requestedReview);
```

Comment:
consider renaming to isReviewRequested / consistent with booleans like isDialogVisible / isEditable 💄